eng/pipelines/coreclr/jit-offbydefault-extra.yml

```diff
@@ -3,8 +3,8 @@ trigger: none
 pr: none
 
 schedules:
-- cron: "0 10 * * 6,0"
-  displayName: Sat and Sun at 2:00 AM (UTC-8:00)
+- cron: "0 22 * * 0"
+  displayName: Sun at 2:00 PM (UTC-8:00)
```

Comment:
IMO, it's useful for it to run both Saturday and Sunday. Not because we need both runs, but because you're less likely for the weekly testing to be completely torpedoed by a single infrastructure failure.